### PR TITLE
Clarify dev environment

### PR DESCRIPTION
**Pull request by evrardjp**
The Makefile is only used for testing purposes, and
is not used in packaging. It most likely won't change
anytime soon.

The `make install` instruction can go away:
For test purposes, we have tox (and make test).
We should probably not install things on the host
without a venv anyway with `make install` (at best
we should use `pip install -e .`).

The `make test` uses tox to install the software, and
should stay.

The other two make instructions are currently broken,
and exposed through skuba itself, so I don't expect
this to be required.

Next to this, the README file had to be updated.

### CODE REVIEW

**Comment by evrardjp** [comment]
Acceptance test is still blocked by https://github.com/SUSE/skuba/pull/1066

**Comment by evrardjp** [comment]
Rebased.

File: skuba-update/Makefile [hunk 1]
```diff
@@ -1,17 +1,5 @@
 .PHONY: all
-all: install
-
-.PHONY: install
-install:
-	./setup.py install
```
Comment (cmurphy) [inline skuba-update/Makefile:6]: If developers like using `make` as the interface for installing why change that?

Comment (evrardjp) [reply skuba-update/Makefile:6]: because that's the wrong command?

suse-package/suse-changelog are not working, proving it's not used by devs, and has rotten (it's a duplicate from the time skuba and skuba-update were separate).

For install, I suppose ``pip install -e .`` should be used instead ?

**Comment by evrardjp** [comment]
> The commit message/PR description could be clarified slightly, it currently says `The Makefile...` and I first assumed it was referring to the top-level Makefile.

Done.

File: skuba-update/Makefile (hunk 1 above)
Comment (Klaven) [inline skuba-update/Makefile:6]: my question here, is this file still needed? I am not super famillier with the project but it did not seem like this was used elsewhere? can we also delete setup.py while we are at it?

File: skuba-update/README.md [hunk 2]
```diff
@@ -18,20 +18,14 @@ To set the development environment consider the following commands:
...
-# Install development dependencies
-$ pip install -r dev-requirements.txt
-
-# We need the shellcheck utility. You can install it from the package manager
-# you might be using.
-$ sudo zypper install ShellCheck
-
-# Run tests and code style checks
+# Run tests and code style checks without containers
 $ make test
```
Comment (flavio) [inline skuba-update/README.md:29]: Maybe something to classify under "tech debt": why don't we provide a make target that builds a container with all the test dependencies and another make target that runs the tests inside of this container?

This is a common approach used by lots of projects

Comment (Klaven) [reply skuba-update/README.md:29]: :+1:

**Comment by evrardjp** [comment]
Well, if nobody cares, I will just close this.
